### PR TITLE
enh(project): reworked everything

### LINKED ISSUE #35: build model for age estimation
use classification
modify our model, so it predcicts age
use parameters from https://www.vision.ee.ethz.ch/publications/papers/articles/eth_biwi_01299.pdf

## usefull links:
- https://keras.io/callbacks/

### PR DESCRIPTION
**Pull request by mooxl**
for the purpose of compiling i reworked our cnn, because there were some inferences with our vgg16 and the pretrained weights in the file. Because of that i used the preimplemented VGG16 from Keras and added 4 layers beneath it. There was also a bug in using the softmax activation in the last layer of our gendermodel, which affected the accuracy and the loss of our network. To fix that i have used the sigmoid activation function. I put everything in the main, for faster development. The learning rate of either the dense layer or the convolutional layer were reworked and are like they should be. I would suggest we write the train method together with the imagegroup because i dont know how their dataloader is working. btw its compiling now --- Closes #35 #38

### CODE REVIEW

**Comment by mooxl** [comment]
i you want to have the old structure of the project (with classes), we could do that after we wrote the trainmethod
